views/peeps/index.erb

```diff
@@ -0,0 +1,8 @@
+<h1>"Welcome to Chitter!"</h1>
+<form action='/sign_up' method='get'>
+  <button name='Sign up' type='submit'>Sign up</button>
+</form>
+
+<form action='/peeps' method='get'>
+  <button name='showpeeps' type='submit'>See peeps</button>
```

Comment:
Good navigation, might want a back button or a way to get back to this page without typing.